generator/wrapper_gen.py

```diff
@@ -81,15 +83,10 @@ class pdDataFrame:
     class pdSeries:
         pass
 
-try:
-    from modin import pandas
-    mdDataFrame = pandas.DataFrame
-    mdSeries = pandas.Series
-except ImportError:
-    class mdDataFrame:
-        pass
-    class mdSeries:
-        pass
+if "modin" in sys.modules:
+    from modin import pandas as mdpd
+    mdDataFrame = mdpd.DataFrame
+    mdSeries = mdpd.Series
```

Comment:
Are they local or global variables? Are they alive after leaving "if" statement?

Reply:
I agree with Egor, there is no need for a global search

Reply:
Agree. I have replaced it already